dateparser/timezones.py

```diff
@@ -6,7 +6,7 @@
 timezone_info_list = [
     {
         'regex_patterns':
-            [r'(\W|\d|_)\(?%s\)?$'],
+            [r'(\W|\d|_)\(?%s(\)|$|[^\w+])'],
```

Comment:
Since, braces don't provide any contextual meaning I believe we can strip them off the date string at the time of extracting timezone. What benefits do you imagine we will have by moving timezone extraction to a level higher?